### DISCUSSION TITLE
Upload a same name .png file but no effect

### DISCUSSION THREAD

**Discussion by victoryminers** [community/community#31009] [Other Feature Feedback, Questions, & Ideas]
I upload a new .png as same name   logo.png  , but it's not change.

**Comment by radar-blip** [comment]
Sometimes it takes a minute, check again. Also, refresh the page while you're there.

**Reply by victoryminers** [reply]
Still no work after 30mins.

**Reply by victoryminers** [reply]
I had tried upload  repeat , it was also no effect.

**Reply by PossibleReference** [reply]
Try deleting then uploading the png

**Reply by victoryminers** [reply]
> Try deleting then uploading the png
Also no effect for me.

**Reply by radar-blip** [reply]
Maybe rename it to something else.

**Reply by victoryminers** [reply]
> Maybe rename it to something else.

Lol, i delete .git  then  add a new project as same name,  it's alse no effect of the .png.

**Reply by radar-blip** [reply]
I give up, I dont know.

**Reply by victoryminers** [reply]
It's work for me today.

**Reply by PossibleReference** [reply]
Make sure to mark the solution as an answer to help out everyone else!

**Comment by PossibleReference** [comment]
This worked for me, however it was an HTML file. Could pngs not be fully supported with this feature yet?

**Reply by radar-blip** [reply]
pngs always worked just fine for me.

**Comment by jfullstackdev** [comment]
what you might be seeing is a cached data, try logging in in Incognito mode to see whether it's still the same thing or not

**Reply by victoryminers** [reply]
How to log in in Incognito mode?

**Reply by jfullstackdev** [reply]
is it Chrome ? look at the three dots upper right corner of the window, then there you have it 

![image](https://user-images.githubusercontent.com/47092464/187033540-efeb376f-a9e5-4dcd-bb62-b4c4a516e1f3.png)

**Reply by victoryminers** [reply]
done

**Reply by jfullstackdev** [reply]
is the page now updated ? or the same as the regular tab ?

**Reply by victoryminers** [reply]
Yes , it was updated automatic.

**Comment by yangjianlin** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
